R/R/flags.R

```diff
@@ -38,6 +38,26 @@ parse_arguments <- function(arguments = NULL) {
       }
       next
     }
+
+    if (argument == "step-functions"){
+      i <- i + 1
+      if (i <= n){
+        if (arguments[i] == "trigger"){
+          i <- i + 1
+          if (i <= n){
+            values$step_functions <- paste("trigger", arguments[i])
```

Comment:
What's the rationale for this conditional?

Reply:
Misread the trigger options. Will remove the conditional and submit a change.